src/liveevent/liveevent.controller.ts

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line no-redeclare
+import { Body, Controller, Delete, Get, Param, Post, Put } from '@nestjs/common';
+import { ApiOkResponse, ApiOperation } from '@nestjs/swagger';
+import { LiveEvent } from '@prisma/client';
+import { CreateEventDto, UpdateEventDto } from './dto/liveEvent.sto';
+import { LiveEventEntity, LiveEventWithIdEntity, SeparationEventListEntity } from './entity/liveEvent.entity';
+import { LiveeventService } from './liveevent.service';
+import { SeparationEventList } from './types';
+
+@Controller('liveevent')
+export class LiveeventController {
+   constructor(private readonly service: LiveeventService) {}
+
+   @Get()
+   @ApiOperation({ summary: '全件取得' })
+   @ApiOkResponse({ type: SeparationEventListEntity })
+   async getAllEvent(): Promise<SeparationEventList | {}> {
```

Comment:
ここ何もなかったら空オブジェクトが返されるって感じ？

```suggestion
   async getAllEvent(): Promise<SeparationEventList | {}> {
```

Reply:
そう　swagger対応させてないかも